package testcases.CWE190_Integer_Overflow.s06;
import testcasesupport.*;
import javax.servlet.http.*;
import java.security.SecureRandom;
public class J16902 extends AbstractTestCase {
    private void badSink(int data) throws Throwable {
        data++;
        int result = (int) (data);
        IO.writeLine("result: " + result);
    }
    public void handle() throws Throwable {
        int data;
        data = (new SecureRandom()).nextInt();
        badSink(data);
    }
    public static void main(String[] args) throws ClassNotFoundException,
            InstantiationException, IllegalAccessException {
        mainFromParent(args);
    }
}
